Example.ipynb

```diff
@@ -112,7 +109,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
```

Comment:
Before commiting file, either clear inputs or rerun it, so that either cell numbers are sequential or there is no output at all.